static/css/bureau-structure.less

```diff
@@ -54,18 +59,18 @@
             padding-top: 45px;
             border-width: 0;
 
-            &:before, 
+            &:before,
             &:after {
                 content: '';
-                position: absolute; 
-                top: 0; 
-                width: 50%; 
+                position: absolute;
+                top: 0;
+                width: 50%;
                 height: unit(@grid_gutter-width, px);
```

Comment:
[`@grid_gutter-width`](https://github.com/cfpb/cf-grid/blob/gh-pages/src/cf-grid.less#L42) is already in pixels.